tsdevops/npm-lerna/assets/setup_env.sh

```diff
@@ -50,7 +52,7 @@ curl -X POST \
     -u "${USERNAME}:${PASSWORD}"
 
 # init local repo
-mkdir -p "${GIT_DIR}"
+mkdir -p "${GIT_DIR}/${REPO_NAME}"
```

Comment:
Thanks a lot! I remove this line again because this would break the git clone some lines below.

Reply:
somehow the folder creation process worked yesterday

Reply:
and there this line was not existing